### PR TITLE
FAIR Visualisation in Organisation and DMP pages

### PR DESCRIPTION
**Pull request by kjgarza**
## Purpose
Product Spec  for FAIR Visualisation in Organisation and DMP pages

closes: _Add github issue that originated this PR_

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

File: product_specifications/FBF44gf/readme.md [hunk 1]
```diff
@@ -0,0 +1,582 @@
+# FAIR Visualisation in Organisation and DMP pages
+
+## Problem
+
+Development of a dashboard and further extension GraphQL API nodes building on connections made possible via the networked DMP that allows users to track connections between DMPs, investigators, outputs, organizations, research methods and protocols; and display citations throughout the research lifecycle.
```
Comment (sarala) [inline product_specifications/FBF44gf/readme.md]: This is a long sentence and not quite clear. I think I understand what you are trying to say, but can you break it into a few sentences.

File: product_specifications/FBF44gf/readme.md [hunk 2]
```diff
@@ -0,0 +1,581 @@
...
+
+
+Problem validation: [A workshop](https://miro.com/app/board/uXjVOZtNNlI=/?share_link_id=611105669245) was carried out to validate the problem. We used a lighting workshop format to identify goals and challenges. Then the data was analyzed in terms of a data visualization framework and user stories were drawn from there. In a secondary call, we sat with all PIs and project leads a prioritized the [user stories](https://docs.google.com/document/d/1lJ6J-OuKiVSMTR_AoEVg_LirjufTBkAwqR4XbPfDQYA/edit).
+
+
+## Solution 
+
+To help administrators of organizations and data management plans in their activities to prepare materials to demonstrate the impact of a collection of work Datacite commons would include a set of features that would allow users to get access to all related content to ROR-ids and DMP-ids.
```
Comment (sarala) [inline product_specifications/FBF44gf/readme.md]: "," before DataCite Commons

File: product_specifications/FBF44gf/readme.md [hunk 3]
```diff
@@ -0,0 +1,581 @@
...
+Problem validation: [A workshop](https://miro.com/app/board/uXjVOZtNNlI=/?share_link_id=611105669245) was carried out to validate the problem. We used a lighting workshop format to identify goals and challenges. Then the data was analyzed in terms of a data visualization framework and user stories were drawn from there. In a secondary call, we sat with all PIs and project leads a prioritized the [user stories](https://docs.google.com/document/d/1lJ6J-OuKiVSMTR_AoEVg_LirjufTBkAwqR4XbPfDQYA/edit).
+
+
+## Solution 
+
+To help administrators of organizations and data management plans in their activities to prepare materials to demonstrate the impact of a collection of work Datacite commons would include a set of features that would allow users to get access to all related content to ROR-ids and DMP-ids. 
+
+Solution Validation: We conducted two validation sessions:
+1. There was a validation session with end users of [DataCite Commons]()
+2. There was a validation with stakeholders of [Datacite services]()
```
Comment (sarala) [inline product_specifications/FBF44gf/readme.md:17]: These links are linked to this page.

File: product_specifications/FBF44gf/readme.md [hunk 4]
```diff
@@ -0,0 +1,581 @@
...
+### Detailed User Stories
+
+Detail the user stories providing the acceptance criteria. All user stories are ordered by priority. 
+
+
+```cucumber
+
+
+
+Feature: Getting all scholarly outputs linked to an organisation
```
Comment (sarala) [inline product_specifications/FBF44gf/readme.md:31]: I found the scenario's a little confusing here. The results of all scenario's should appear in this feature?

File: product_specifications/FBF44gf/readme.md [hunk 5]
```diff
@@ -0,0 +1,581 @@
...
+        Scenario: Getting all scholarly outputs linked to a DMP linked to an organisation
+
+            Given there exist DOIs with relatedIdentifierType equals "OutputManagementPlan",
+              And the relatedIdentifier DMP has contributor.contributorType equals "Sponsor",
+              And the relatedIdentifier DMP has relatedIdentifier contributor.nameIdentifier equals Organisation's ROR-Id,
+             When that the user is in a Organisation Page
+             Then the page should display a list of the  DOIs
+
+
+Feature: Export all scholarly outputs linked to an organisation
```
Comment (sarala) [inline product_specifications/FBF44gf/readme.md:68]: How do you select the 500? Is it the most recent or random?

**Review by sarala — COMMENTED** [review]
Can we also break it into 2 phases where we do the simple visualisations and the reports, and the rest for the second phase.

**Comment by kjgarza** [comment]
> Can we also break it into 2 phases where we do the simple visualisations and the reports, and the rest for the second phase.

I think when we get to planning one can break it in as many phases as one finds suitable

File: product_specifications/FBF44gf/readme.md (hunk 5 above)
Comment (kjgarza) [reply product_specifications/FBF44gf/readme.md:68]: most recent. I think I might add that they should be sorted by publication year.

File: product_specifications/FBF44gf/readme.md (hunk 4 above)
Comment (kjgarza) [reply product_specifications/FBF44gf/readme.md:31]: yes indeed. This is primarily a limitation of the gherkin language. ORs are not supported, but generally, there is an OR between every scenario. Maybe I can add a note on that.

File: product_specifications/FBF44gf/readme.md (hunk 3 above)
Comment (kjgarza) [reply product_specifications/FBF44gf/readme.md:17]: We are yet to run the validations.

**Pull request merged**